public/javascripts/studentControllers.js

```diff
@@ -28,27 +25,23 @@ var studentController = function($scope,  $http, $state) {
 
 var addDailyEntryController = function($scope, $http, $location) {
 
-  // Ititialize the datepicker with settings
-  $('.datepicker').datepicker({
-    format: 'mm/dd/yyyy',
-    autoclose: true,
-    todayHighlight: true
-  });
-
+  $scope.popup1 = {
+    opened: false
+  };
 
-   $scope.slider = {
-      value: 3,
-      options: {
-          showSelectionBar: true,
-          getPointerColor: function(value) {
-              return '#308a83';
-          },
-          getSelectionBarColor: function(value) {
-            return '#eee';
-          }
-      }
+  $scope.slider = {
+    value: 3,
+    options: {
+      showSelectionBar: true,
+      getPointerColor: function(value) { return '#308a83'; },
+      getSelectionBarColor: function(value) { return '#eee'; }
+    }
   };
 
+  $scope.newDailyEntry = {engageContent: 5, engagePeer: 5, engageAdult: 5};
```

Comment:
I didn't make them all fives, I forget who did...I think it was just to initialize the sliders with values, which may or may not be a good thing. Thoughts?